mesop/labs/chat.py

```diff
@@ -200,6 +202,8 @@ def submit():
         yield
     state.in_progress = False
     yield
+    me.focus_component(key=f"{len(state.output)}")
```

Comment:
I would add a prefix to this key like "input-" just to make it clearer for people who read this code what it's doing.